renv/.gitignore

```diff
@@ -0,0 +1,6 @@
+library/
```

Comment:
you got one gitignore in package directory and one here, how do they differ from eachother?

Reply:
Not sure if it is correct, but as I search through repos that is quite common, that .gitignore files are in root as well as in renv and .github folders. As I read, keeping .gitignore in subfolder makes sense in that way all the file-rules apply to the root directory of .gitignore file. https://stackoverflow.com/questions/53208235/multiple-gitignore-in-subfolders
Can we keep it that way or do you have other view?